frontend/src/features/leases/detail/LeasePages/LeasePageForm.tsx

```diff
@@ -25,39 +27,44 @@ export const LeasePageForm: React.FunctionComponent<ILeasePageFormProps> = ({
   refreshLease,
   setLease,
   children,
-  onUpdate,
 }) => {
+  const location = useLocation();
+  const { edit } = queryString.parse(location.search);
   return (
     <StyledLeasePage>
       <StyledLeasePageHeader>
         <Styled.LeaseH2>{leasePage.header ?? leasePage.title}</Styled.LeaseH2>
         {leasePage.description && <p>{leasePage.description}</p>}
       </StyledLeasePageHeader>
-      <Formik<IFormLease>
-        initialValues={{
-          ...defaultFormLease,
-          ...apiLeaseToFormLease(lease),
-        }}
-        enableReinitialize={true}
-        onSubmit={async (values, { setSubmitting }) => {
-          try {
-            if (!leasePage.subRoute) {
+      {!edit ? (
+        <Formik<IFormLease>
+          initialValues={
+            leasePage.title !== leasePages.get(LeasePageNames.DETAILS)?.title
+              ? ({
+                  ...defaultFormLease,
+                  ...apiLeaseToFormLease(lease),
+                } as IFormLease)
+              : { ...defaultFormLease, ...apiLeaseToFormLease(lease) }
```

Comment:
hmm, could you please comment on this logic a bit? It seems both code paths do the same here...

Reply:
Oh yes, this was a side effect of deciding to simplify the lease page code by putting the lease in a context. I should be able to just have
{ ...defaultFormLease, ...apiLeaseToFormLease(lease) } 
now